watcher/src/watcher_db.rs

```diff
@@ -445,15 +446,8 @@ impl WatcherDB {
 
     /// Get the timestamp from the watcher, or an error code,
     /// using retries if the watcher fell behind
-    /// Fog ledger server db fetcher and Ingest Server worker are both affected
-    /// by this change to move get_watcher_timestamp to a shared crate in mc
-    /// watcher
-    pub fn get_watcher_timestamp(
-        &self,
-        block_index: BlockIndex,
-        watcher_timeout: Duration,
-        logger: &Logger,
-    ) -> u64 {
+    /// The block index from ingest or ledger for example, watcher timeout set
```

Comment:
What is "watcher timeout set"?

Reply:
please see watcher timeout set is  the time that has elapsed to indicate watcher is behind in the ingest or ledger for example

Reply:
This shouldn't be explained to me over a comment on the PR, this should be in the file so that everyone that reads it in the future can understand what the parameter is for.

Reply:
ok will add comment in the file